infra/pr-bot/aggregate-size-plugin.js

```diff
@@ -18,14 +22,50 @@ class AggregateSizePlugin extends PluginInterface {
       `workbox-cacheable-response`,
       `workbox-cache-expiration`,
     ];
-    const files = glob.sync(path.posix.join(
-      afterPath, 'packages', `(${packagesToAggregate.join(',')})`,
+    const globPattern = path.posix.join(
+      afterPath, 'packages', `{${packagesToAggregate.join(',')}}`,
       'build', 'browser-bundles', '*.production.js',
-    ));
-    console.log(files);
+    );
+    const files = glob.sync(globPattern);
+    let totalSize = 0;
+    files.forEach((filePath) => {
+      const stats = fs.statSync(filePath);
+      totalSize += stats.size;
+    });
+
+    const percentValue = (totalSize / MAX_SIZE) * 100;
+    const percentString = parseFloat(percentValue).toFixed(0);
+
+    let totalSizeString = totalSize.toString();
+    let unitString = 'B';
+    if (totalSize > 1000) {
+      unitString = 'KB';
+      totalSizeString = parseFloat(totalSize).toFixed(2);
+    }
```

Comment:
Fwiw, modules like https://www.npmjs.com/package/bytes allow you to handle size formatting pretty cleanly and accept byte input like you're using for the totalSize above.